XcodeServerSDKTests/TestUtils.swift

```diff
@@ -56,4 +58,87 @@ extension XCTestCase {
         let configuration = bot.configuration
         return configuration
     }
+}
+
+// MARK: Exception assertions
+// Based on: https://forums.developer.apple.com/thread/5824
+extension XCTestCase {
+    /**
+    Replacement method for XCTAssertThrowsError which isn't currently supported.
+    
+    - parameter message: Message which should be displayed
+    - parameter file:    File in which assertion happened
+    - parameter line:    Line in which assertion happened
+    - parameter block:   Block of code against which assertion should be matched
+    */
+    func XCTempAssertThrowsError(message: String = "", file: String = __FILE__, line: UInt = __LINE__, _ block: () throws -> ()) {
+        do {
+            try block()
+            
+            let msg = (message == "") ? "Tested block did not throw error as expected." : message
+            XCTFail(msg, file: file, line: line)
+        } catch {}
+    }
+    
+    /**
+    Replacement method for XCTAssertThrowsSpecificError which isn't currently supported.
+    
+    - parameter kind:    ErrorType which is expected to be thrown from block
+    - parameter message: Message which should be displayed
+    - parameter file:    File in which assertion happened
+    - parameter line:    Line in which assertion happened
+    - parameter block:   Block of code against which assertion should be matched
+    */
+    func XCTempAssertThrowsSpecificError(kind: ErrorType, _ message: String = "", file: String = __FILE__, line: UInt = __LINE__, _ block: () throws -> ()) {
+        do {
+            try block()
+            
+            let msg = (message == "") ? "Tested block did not throw expected \(kind) error." : message
+            XCTFail(msg, file: file, line: line)
+        } catch let error as NSError {
+            let expected = kind as NSError
+            if ((error.domain != expected.domain) || (error.code != expected.code)) {
```

Comment:
I know it can be downcasted; the fact that it can be done doesn't mean its a good idea.

Also; note that I'm not complaining about the solution :+1: 
I'm grateful by how this implementation looks TBH honest its cleaner that something I would have coded :sweat_smile: 

I just feel like having to bridge shows there's an issue with the model as it is now.

I think this is `radar` worthy TBH because  should provide an assert out of the :package: .

If you end up `radar`ing it I'll dupe it in no time :wink: